project/ProjectAutoPlugin.scala

```diff
@@ -106,8 +106,8 @@ object ProjectAutoPlugin extends AutoPlugin {
  )
 
   def determineMimaPreviousArtifacts(scalaBinVersion: String): Set[ModuleID] = {
-    val compatVersions: Set[String] = if (scalaBinVersion.startsWith("2.13")) Set.empty else {
-      Set("3.5.0")
+    val compatVersions: Set[String] = if (scalaBinVersion.startsWith("2.13")) Set("3.5.2") else {
+      Set("3.5.0", "3.5.1", "3.5.2")
```

Comment:
We can simplify it by only checking against 3.5.2. Since 3.5.0 and 3.5.1 are binary compatible with 3.5.2. 

This can be on another PR